frontend/app/src/components/defi/Lending.vue

```diff
@@ -47,6 +47,9 @@
         <stat-card title="DAI earned" :locked="!premium">
           {{ accountGain(selection.address).dp(decimals) }}
         </stat-card>
+        <stat-card title="approximate $ earned" :locked="!premium">
+          {{ accountGainUsdValue(selection.address).dp(decimals) }}
```

Comment:
wouldn't this make sense to become an `<amount-display/>`? we want it to change with the selected currency right?

Reply:
Probably ... I really have no idea what that is. I just copied pasted stuff to expose the usd value in the frontend. Did not even run it.

Reply:
I will pick up fixing these. Do we want to push this to develop so I can rebase on top? I already have some changes made in my PR feat_makervault_improvements for Lending as well.

Reply:
@isidorosp We could, but this will break develop. Linting is not even fine here. But I also guess the loans/vaults may look wrong. How fast would you be able to adjust them?

Reply:
@isidorosp okay so fixed linting and build. Merging is your call or not. Let me know.

Reply:
The backend stuff is all up to you, the frontend stuff @kelsos and I can fix for sure so no problem on my end. @kelsos ?

Reply:
Backend should be done here. If something is missing we can add it later. So you okay with me merging then.